src/doom/am_map.c

```diff
@@ -1681,12 +1681,12 @@ static void AM_rotatePoint (mpoint_t *pt)
     pt->x -= mapcenter.x;
     pt->y -= mapcenter.y;
 
-    tmpx = (int64_t)FixedMul(pt->x, finecosine[mapangle>>ANGLETOFINESHIFT])
-         - (int64_t)FixedMul(pt->y, finesine[mapangle>>ANGLETOFINESHIFT])
+    tmpx = (int64_t)FixedMul(pt->x, finecosine[(ANG90 - viewangle)>>ANGLETOFINESHIFT])
```

Comment:
Could you turn this onto a `const angle_t actualangle = ANG90 - viewangle` instead of recalculating it four times, please?

Reply:
Sure, gladly. Done in https://github.com/fabiangreffrath/crispy-doom/pull/734/commits/fa66e7f1d17b752d1215f4b4544dca44fb360bf9.